src/main/resources/documentation/domain-model-overview.md

```diff
@@ -0,0 +1,344 @@
+
++ **SocialGroups** are the societal or cultural organizations that the people in the area of study are a part of. Examples of SocialGroups are church groups, families, book clubs, etc.
+
+The **Relation Entities** are Membership, Relationship, and Residency. 
+
++ **Memberships** are what tie an Individual and a SocialGroup together; a Membership is made for every SocialGroup a person a part of. An example of a Membership is being a member of a particular family. 
+
++ **Relationships** are what tie two Individuals together; a Relationship can be made for each way an individual is related to anyone else. An example of a Relationship is being the son of someone, being the pastor of someone, or being someone’s lawyer.
+
++ **Residencies** are what tie Locations and Individuals together; a Residency is made for each Location an Individual is a resident at. An example of a Residency is being a resident at your house (a Location) and perhaps your parent’s house (A different Location).
+
+Without explaining the Update Events yet, a picture is starting to form on how the Census and Relation Entities are oriented with respect to one another. To give things more context, however, it is necessary to explain the FieldWorker entity. The OpenHDS is a system that relies on data being collected in an area of study. To help keep track of who collected what and when, the FieldWorker is needed. All of the entities in *figure 2* are collected either directly or indirectly by a FieldWorker and thus all are dependent on the FieldWorker. Without Update Events, here is a look at how things are dependent on each other…
+
+#####How things fit together (with no Update Events).
+![Figure 3](figure-3.png)
+*figure 3*
+
+This image makes it easier to visualize how things fit together, but maybe an example would be more helpful…
+
+#####A real-world fake example.
+![Figure 4](figure-4.png)
+*figure 4*
+
+There are three Individuals: Bill Peterson, Joe Smith, and Jen Smith. 
+
+Bill has a Residency at the Location 32 Main st, his home. 
+
+Bill also has a Membership for the SocialGroup ‘Teacher’s Union’.
+
+Bill’s house, the Location 32 Main st, is a child of the LocationHierarchy Main St. 
+
+It is easy to imagine that this Main St. could be the same one branching off of the City of Bangor in Figure 1, which would make Bill a resident of the State of Maine, USA. 
+
+On this same street, but at a different Location, lives Joe and Jen Smith. 
+
+The Smiths are a family, so they both have a Membership for the SocialGroup ‘Smith Family’. 
+
+The Smiths also share a Relationship each, one from each person’s point of view: Jen is Joe’s wife, and Joe is Jen’s husband. 
+
+It is easy to imagine a FieldWorker walking down Main St. and knocking on Bill Peterson’s door and surveying him and then continuing up the street to the Smith Family’s door and collecting all this information.
+
+#### The Update Events.

```

Comment:
I think it might help to lead with the idea of a Visit.  "A FieldWorker visits a location again and records that things have changed, for example..."